assets/js/blocks/products/all-products/edit.tsx

```diff
@@ -193,6 +204,7 @@ class Editor extends Component {
 			template: this.props.attributes.layoutConfig,
 			templateLock: false,
 			allowedBlocks: Object.keys( this.blockMap ),
+			renderAppender: true,
```

Comment:
Could you say more about this change? I think `renderAppender` should be either `false` or a `Component` instance according to [docs](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/inner-blocks/README.md#renderappender) and following them:

> Default: - undefined. When renderAppender is not specified, the default appender is shown. If a false value is provided, no appender is rendered.

In this case lack of `renderAppender` was expected, hence I'm curious about this change.

Reply:
@kmanijak you're correct, I've fixed that.